setup.rst

```diff
@@ -99,6 +105,13 @@ Open your browser and navigate to ``http://localhost:8000/``. If everything is
 working, you'll see a welcome page. Later, when you are finished working, stop
 the server by pressing ``Ctrl+C`` from your terminal.
 
+.. note::
+
+    Locally, you can also use the PHP built-in webserver to run Symfony using
+    ``php -S localhost:8000 -t public``. This however does not provide most of
+    the features of the Symfony local web server or a full Nginx or Apache
+    set-up and can result in slow page builds.
```

Comment:
Maybe we should mention one or two features that are lacking to better help decide if they are needed.
```suggestion
    set-up and can result in slow page builds, such as HTTPS with a signed certificate.
```